docs/user/devops-pipelines.md

```diff
@@ -52,3 +51,78 @@ will be copied over**_. Values must be set on the next Stage's Instance explicit
 **Instance Settings** - None of your Instance Settings will be copied over (e.g. Editor, Palette or Security Settings).
 
 This ensures a split between your staging environments.
+
+## Pipeline Stage details
```

Comment:
As an engineer I like this section, though difficult to parse. No changes required, I believe almost all specs are